infretis/core/tis.py

```diff
@@ -49,7 +59,15 @@ def log_mdlogs(inp: str) -> None:
 
 
 def run_md(md_items: dict[str, Any]) -> dict[str, Any]:
-    """run md function."""
+    """Execute MD simulation step(s).
```

Comment:
Technically it executes any of the shooting moves that require md. I am not sure how to name this function as run_mc does not technically work since we do (inf swap elsewhere)